config/longrun_configs/slabplanet_aqua_evolve_ocean.yml

```diff
@@ -1,4 +1,4 @@
-atmos_config_file: "config/longrun_configs/longrun_aquaplanet_allsky_0M.yml"
+atmos_config_file: "config/longrun_configs/amip_target_edonly.yml"
```

Comment:
Same here as above (aqua_evolve_ocean might suggest the aquaplanet config file, and the corresponding tomls)